examples/material_gallery/test/example_code_parser_test.dart

```diff
@@ -0,0 +1,77 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+
+import 'package:mojo/core.dart' as core;
+
+import 'package:test/test.dart';
+
+import '../lib/gallery/example_code_parser.dart';
+
+void main() {
+  test('Material Gallery example code parser test', () {
+    testWidgets((WidgetTester tester) {
+      TestParsingWidget testWidget = new TestParsingWidget();
```

Comment:
The expect calls are being executed (tested that). You are right, won't need the widget, fixing that. Will test async calls and see if I can get that to work!